controllers/categories.controller.js

```diff
@@ -1,7 +1,7 @@
 const { fetchCategories } = require("../models/categories.model");
 
-exports.getCategories = (req, res) => {
-  fetchCategories().then((category) => {
-    res.status(200).send(category);
-  });
+exports.getCategories = (req, res, next) => {
+  fetchCategories()
+    .then((category) => res.status(200).send(category))
```

Comment:
I would suggest changing this to the key of 'Categories', as category is singular and could cause confusion as to what data we are sending back